stock_available_to_promise_release/models/stock_move.py

```diff
@@ -49,26 +49,47 @@ def _ordered_available_to_promise(self):
         if not self._should_compute_ordered_available_to_promise():
             return 0.0
         available = self.product_id.with_context(
-            location=self.warehouse_id.lot_stock_id.id
-        ).virtual_available
+            **self._order_available_to_promise_qty_ctx()
+        ).qty_available
         return max(
             min(available - self._previous_promised_qty(), self.product_qty), 0.0
         )
 
+    def _order_available_to_promise_qty_ctx(self):
+        return {
+            "location": self.warehouse_id.lot_stock_id.id,
```

Comment:
Maybe add a comment to explain this context is used in `_get_domain_locations`, took me some time to figure it out.